docs/agile/scrum.md

```diff
@@ -9,12 +9,14 @@ Instead of using sprint names like Sprint 1, Sprint 2, and so on try to name 202
 - We can [sort by string format of the date](/general-rules-and-definitions.md/#sort-by-string-format-of-the-date)
 
 ## Refinement
+The meeting is where we discuss tasks and estimate them. It is important to have easy to find one source of truth with a list of the refinements.
```

Comment:
**suggestion** Maybe it is worth to add one sentence about the purpose of the meeting. Like increase the transparency, and further specification of the items.